docs/products/kafka/kafka-mirrormaker/getting-started.rst

```diff
@@ -0,0 +1,47 @@
+Getting started
+===============
+
+Aiven services are managed in the Aiven `web console <https://console.aiven.io/>`__ . When you first log into the console with your email address and password, you will see the **Services** view, which shows you all the services in the currently selected project.
+
+.. Warning::
+
+    Aiven for Apache Kafka MirrorMaker2 services can be created only alongside at least one existing Aiven for Apache Kafka running services. 
```

Comment:
At the end: .... running service

(to match the beginning of the sentence: at least one ..... running service)

Reply:
Well spotted, thanks!